proposals/csharp-11.0/low-level-struct-improvements.md

```diff
@@ -411,6 +410,30 @@ Span<int> ComplexScopedRefExample(scoped ref Span<int> span)
 }
 ```
 
+#### Rules for object initializers
+
+The *safe-context* of an object initializer expression is narrowest of:
+
+1. The *safe-context* of the constructor call.
+2. The *safe-context* arguments to member initializer indexers that can escape to the receiver (can escape to *caller-context*)
```

Comment:
Should this be *ref-safe-context* of `ref` arguments to indexers (specifically, an `in` argument)?

Reply:
This is a case of practically speaking no. The `in` part of an indexer can't escape into `this` because it's return only. Hmm ... guess you could if it was `[UnscopedRef]`. Not sure if we have good testing for that. The 99% case is the value is what matters. 

Spec wise can say both thogugh.